client/signup/controller.js

```diff
@@ -305,6 +308,38 @@ export default {
 
 		next();
 	},
+	setSelectedSiteForSignup( { store: signupStore, query }, next ) {
+		const { getState, dispatch } = signupStore;
+		const signupDependencies = getSignupDependencyStore( getState() );
+
+		const siteFragment = signupDependencies?.siteSlug || query?.siteSlug;
```

Comment:
Why did we call that variable `siteFragment` instead of `siteSlug` (given how we retrieve the value)?